discover/deployment/articles-dxp/05-maintaining-liferay/01-patching-liferay-dxp/02-using-the-patching-tool/02-working-with-patches.markdown

```diff
@@ -71,8 +72,22 @@ information for the `diff` command. Here are the `store` command options:
 - `update`: Adds or updates patch level information.
 - `rm`: Removes previously stored patch level information.
 
+
 For detailed usage information, run `patching-tool help store`.
 
+## Showing collisions between patches and deployed plugins [](id=list-collisions)
+
+Some patches update files which can be customized through plugins. 
+In these cases the customizations will need to be updated.
+To help identifying which files are affected, run the `patching-tool list-collisions` command.
+This command is an alias for the "diff" command and is equivalent to running the following:
+
+    patching-tool diff collisions files _base
```

Comment:
@gergelyszaz - would you give an example of what the `_base` folder would be?  I'm confused as to whether it would be a subfolder of `patches/` or some folder that contains the customization source code.


Thanks!

Reply:
`_base` is the patch level which will be used for generating the diff.
The `_base` patch level is automatically stored when a patch with source is installed.

Reply:
So if `patching-tool info` returns this …

`Currently installed patches: dxp-3-7110`

… would `patching-tool list-collisions`  be the same as executing this?

`patching-tool list-collisions dxp-3-7110`

Reply:
No. 

dxp-3-7110 would be the name of one installed patch/fix-pack. 
The patch level is managed by running the store command and is like a snapshot of the portal with patches installed. 
E.g.: `patching-tool store update [patch_level_name]`.

At install, if the installable patches contain the source code, the following command is run automatically:
`patching-tool store update _base`
_base is the literal name of the patch level.

`patching-tool list-collisions`
is the literal alias for
`patching-tool diff collisions files _base`